### PR TITLE
added Styling

### PR DESCRIPTION
**Pull request by Reggeditt**
- Added missing JSX elements to project (without adding extra functionality)
- final design was styled to match Zeplin's design (this includes all elements + the same styling)

### CODE REVIEW

File: src/components/addBook.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.add-book-btn {
+  width: 10rem;
+  height: 2.813rem;
+  margin: 1.188rem 0 0 0;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: var(--azure);
+}
```
Comment (binhussen) [inline src/components/addBook.css]: - Kindly, you didn’t stick the font, color, margin, and padding of the given design, Please Update it to fill the requirement and have a better UI 👌
### Given design 👇


### Your design 👇

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Reggeditt , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/addBook.css (hunk 1 above)
Comment (Reggeditt) [reply src/components/addBook.css]: noted. I'll get to wrok on them

Comment (Reggeditt) [reply src/components/addBook.css]: fixed

**Review by BregornOriginal — APPROVED** [review]
### Hi @Reggeditt 

### Your project is complete!  There is nothing else to say other than... it's time to merge it 

### Congratulations! 

allfather-approves-approved

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good work on the setup project 🥇 

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Review by amejid — COMMENTED** [review]
Hi @Reggeditt, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Correct workflow :heavy_check_mark: 
- Good UI :heavy_check_mark: 
- Project is already approved :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**